safehttp/mux.go

```diff
@@ -97,14 +97,38 @@ func NewServeMux(d Dispatcher, domains ...string) *ServeMux {
 
 // Handle registers a handler for the given pattern and method. If another
 // handler is already registered for the same pattern and method, Handle panics.
-func (m *ServeMux) Handle(pattern string, method string, h Handler) {
+//
+// Configs can be optionally passed in order to modify the behavior of the
+// interceptors on a registered handler. Passing a Config whose corresponding
+// Interceptor was not installed will produce no effect.
+func (m *ServeMux) Handle(pattern string, method string, h Handler, configs ...Config) {
+	var changed bool
```

Comment:
Move this to right before the nested for-loop since that is where it is first used.